### ISSUE TITLE
OpenTSDB over secure distributed Hbase

### ISSUE THREAD

**Issue by leosouzadias** [OpenTSDB/opentsdb#884]
I'm trying to configure opentsdb in a secure Hbase, but I could only make it work by doing a "kinit" before starting opentsdb, and after the ticket expires, I need to renew it manually (using a scheduled kinit and restart opentsdb). 
Can't we use keytabs with Asynchbase to have a smoothier behavior?
Another issue is that I can only connect to standalone Hbase, I still can't connect to distributed HBASE, is it supported?
